vmm/src/config.rs

```diff
@@ -2301,6 +2334,45 @@ impl TpmConfig {
     }
 }
 
+impl LandlockConfig {
+    pub const SYNTAX: &'static str = "Landlock parameters \
+        \"path=<path/to/{file/dir}>,access={r,w}\"";
```

Comment:
This looks more correct: `access=[rw]`

Reply:
The existing looks reasonable to me.
@cloud-hypervisor/cloud-hypervisor-maintainers  any thoughts on this?

Reply:
It should be '[rw]'.